examples/api/java/BitVectorsAndArrays.java

```diff
@@ -95,5 +95,6 @@ public static void main(String[] args) throws CVC5ApiException
     System.out.println("The satisfying model is: ");
     System.out.println("  current_array = " + slv.getValue(current_array));
     System.out.println("  current_array[0] = " + slv.getValue(current_array0));
+    slv.close();
```

Comment:
I would suggest (here, and in all other examples) to use the "try-with-resources" idiom instead of calling `close()` manually.

Reply:
Done.